src/components/UserActivityLogs/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+
+function UserActivityLogs() {
+  const [filters, setFilters] = useState({
+    a_user_id: "",
+    start: "",
+    end: "",
+  });
+
+  const [data, setData] = useState([]);
+  const [token, setToken] = useState("");
+
+  const generateApiLink = () => {
+    let link = "http://127.0.0.1:5000/users/activities";
```

Comment:
@khalifan-kfan , I am curious as to why we have this local development link hardcoded for something going to be deployed to staging and then production.
A better implementation would be to use env variables and dynamically assign the value basing on the environment.

Reply:
yes @e-ian , am not sure how that got there, thanks I hadn't noticed, this file let me delete it.